balancer-js/examples/pools/composable-stable/create.ts

```diff
@@ -37,32 +60,22 @@ const rateProviders = [
 ];
 
 const tokenRateCacheDurations = ['0', '0'];
-
 const exemptFromYieldProtocolFeeFlags = [false, false];
-
 const swapFee = '0.01';
+
 const owner = '0x817b6923f3cB53536859b1f01262d0E7f513dB78';
-const factoryAddress = `${ BALANCER_NETWORK_CONFIG[network].addresses.contracts.composableStablePoolFactory }`;
+const factoryAddress = `${BALANCER_NETWORK_CONFIG[network].addresses.contracts.composableStablePoolFactory}`;
 
 async function createComposableStablePool() {
-  // const rpcUrl = `https://mainnet.infura.io/v3/444153f7f8f2499db7be57a11b1f696e`;
-  // const rpcUrl = 'https://goerli.gateway.tenderly.co/4Rzjgxiyt0WELoXRl1312Q'
-  const provider: JsonRpcProvider = new ethers.providers.JsonRpcProvider(
-    rpcUrl,
-    network,
+  await forkSetup(
```

Comment:
Add some comments before this that explains what its doing so if someone copy pasted they would know to remove it.